src/docs/development/platform-integration/c-interop.md

```diff
@@ -0,0 +1,202 @@
+---
+title: "Binding to native code using dart:ffi"
+description: "To use C code in your Flutter program, use the dart:ffi library (currently in preview)."
+---
+
+Flutter mobile can use the [dart:ffi][] library
+to call native C APIs. The _ffi_ acronym stands for
```

Comment:
_FFI_

Reply:
Technically it's an initialism, not an acronym. :) But I'd just say "_FFI_ stands for..."